Common Collections/Vectors/Create a Vector on Your Own/task-info.yaml

```diff
@@ -3,13 +3,16 @@ files:
 - name: src/main.rs
```

Comment:
Maybe we could place lib.rs before main.rs here in yaml, so that when the student opens this task lib.rs would be shown in the code editor instead of main.rs

Reply:
I had a feeling there should be some dependency! Thank you for the insight, did that!